### PR TITLE
Remove invalid todo

### LINKED ISSUE #5178: Remove invalid todo comments
### Acceptance criterial

Todos within the following files should be removed since they are no longer required or have been resolved.

- src/modules/hardwareWallet/components/HardwareWalletStatus/HardwareWalletStatus.js
- src/modules/hardwareWallet/components/HWSigning/HWSigning.js
- src/modules/transaction/utils/transactions.js
- libs/wcm/utils/connectionCreator.js
- setup/react/app/ApplicationBootstrap.js
- libs/wcm/utils/requestHandlers.js & libs/wcm/utils/requestHandlers.test.js
- src/modules/network/components/statistics/statistics.js
- src/modules/transaction/store/actions.js
- src/modules/transaction/utils/transaction.js
- src/utils/ipcLocale.js

### PR DESCRIPTION
**Pull request by eniolam1000752**
### What was the problem?

This PR resolves #5178

### How was it solved?

- Remove deprecated todos

### How was it tested?

- Visually

### CODE REVIEW

**Review by ikem-legend — DISMISSED** [review]

File: libs/wcm/utils/requestHandlers.test.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
-// @todo Add tests by #4418
```
Comment (ManuGowda) [inline libs/wcm/utils/requestHandlers.test.js:1]: Remove this file

File: src/modules/network/components/statistics/statistics.js [hunk 2]
```diff
@@ -261,9 +261,7 @@ const ConnectivityDonutChart = ({ t, connectionData, colorPalette }) => {
 };
 
 const ChartsWithData = ({ networkVersion, height, basic, t }) => {
-  if (!networkVersion) {
-    return null; // @todo Create placeholder
```
Comment (ManuGowda) [inline src/modules/network/components/statistics/statistics.js:265]: the todo was to create placeholder? no changes needed?

**Review by ManuGowda — DISMISSED** [review]

**Review by ManuGowda — DISMISSED** [review]

File: src/modules/blockchainApplication/connection/components/RequestSummary/index.js [hunk 3]
```diff
@@ -155,7 +168,7 @@ const RequestSummary = ({ nextStep, history }) => {
                   metadata: { name: senderAccount?.name, address: senderAccount?.address },
                 }}
                 truncate
-                onSelect={() => {}}
+                onSelect={() => { }}
```
Comment (oskarleonard) [inline src/modules/blockchainApplication/connection/components/RequestSummary/index.js]: why extra space here?

File: src/modules/blockchainApplication/connection/components/RequestSummary/index.test.js [hunk 4]
```diff
@@ -42,9 +41,10 @@ jest.mock('@transaction/utils/transaction', () => ({
 jest.mock('@walletconnect/utils', () => ({
   getSdkError: jest.fn((str) => str),
 }));
-jest.mock('@libs/wcm/utils/requestHandlers', () => ({
-  rejectLiskRequest: jest.fn(),
-}));
+// jest.mock('@libs/wcm/utils/requestHandlers', () => ({
```
Comment (oskarleonard) [inline src/modules/blockchainApplication/connection/components/RequestSummary/index.test.js]: ?

File: src/modules/transaction/store/actions.js [hunk 5]
```diff
@@ -40,54 +40,52 @@ export const resetTransactionResult = () => ({
  */
 export const transactionBroadcasted =
```
Comment (oskarleonard) [inline src/modules/transaction/store/actions.js]: This file is strangely formatted, not correct prettified

**Review by ikem-legend — DISMISSED** [review]

**Review by ikem-legend — APPROVED** [review]

**Review by ManuGowda — APPROVED** [review]

**Review by oskarleonard — APPROVED** [review]

**Pull request merged**